harpy/diagnose.py

```
@@ -0,0 +1,52 @@
+            error = process.stderr.readline()
+            if not output and not error and process.poll() is not None:
+                break
+            if error:
+                rprint(f"[red]{error}", end="", file=sys.stderr)
+            if output:
+                if output.startswith("This was a dry-run"):
+                    process.terminate()
+                    exit(0)
+                else:
+                    rprint(f"[yellow]{output}", end="", file=sys.stderr)
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)